schemas/layer-4.cue

```diff
@@ -1,35 +1,104 @@
 // Top level schema //
 
-"evaluation-plans": [...#EvaluationPlan]
+// Framework Evaluation describes the evaluation of a single framework and its controls.
+#FrameworkEvaluation: {
+    // name of the framework being evaluated, ex: NIST 800-53, CCC, OSPS, etc.
+    // TODO: why materialize/duplicate this data here if a consumer can simply reference the layer 2 data?
```

Comment:
More specifically, should we remove the `name` field altogether from this definition as it is a property of the framework name, not of a FrameworkEvaluation.